subcommand/server-acl-init/command.go

```diff
@@ -11,10 +11,12 @@ import (
 	"sync"
 	"time"
 
+	godiscover "github.com/hashicorp/consul-k8s/helper/go-discover"
 	"github.com/hashicorp/consul-k8s/subcommand"
 	k8sflags "github.com/hashicorp/consul-k8s/subcommand/flags"
 	"github.com/hashicorp/consul/api"
 	"github.com/hashicorp/consul/command/flags"
+	"github.com/hashicorp/go-discover"
```

Comment:
It's used because the command is instantiating some variables types from that library (`discover.Provider`). Golang will fail to compile if there are unused imports, so we would definitely know 😃 

And it looks like if the library is prefixed with `go-`, you can reference it by just mentioning the part after the prefix. This is also the case with `go-hclog`. I didn't know that!